packages/fxa-auth-server/test/local/senders/mjml-emails.ts

```diff
@@ -968,6 +980,45 @@ const TESTS: [string, any][] = [
       { test: 'notInclude', expected: 'utm_source=email' },
     ]],
   ])],
+
+  ['subscriptionPaymentExpiredEmail', new Map<string, Test | any>([
+    ['subject', { test: 'equal', expected: `Credit card for ${MESSAGE.productName} expiring soon` }],
+    ['headers', new Map([
+      ['X-SES-MESSAGE-TAGS', { test: 'equal', expected: sesMessageTagsHeaderValue('subscriptionPaymentExpired') }],
+      ['X-Template-Name', { test: 'equal', expected: 'subscriptionPaymentExpired' }],
+      ['X-Template-Version', { test: 'equal', expected: TEMPLATE_VERSIONS.subscriptionPaymentExpired }],
+    ])],
+    ['html', [
+      { test: 'include', expected: decodeUrl(configHref('subscriptionSettingsUrl', 'subscription-payment-expired', 'update-billing', 'plan_id', 'product_id', 'uid', 'email')) },
+      // { test: 'include', expected: decodeUrl(configHref('subscriptionTermsUrl', 'subscription-payment-expired', 'subscription-terms')) },
```

Comment:
So.. I couldn't find this link in the current/old templates, let alone the new one. Any idea what this is testing?

Reply:
Hmm, looks like in the config it's noted as the cancellation policy URL, do the old templates no longer reference that (I wonder why the old test passes then)?